src/lib/traitify-client.js

```diff
@@ -58,7 +58,14 @@ export default class TraitifyClient {
     } else {
       params = _params;
     }
-
+    // TODO remove conditional after cognitive and interview services
+    // are setup to handle auth from a query param
+    if(url.includes("assessment")) {
+      url += url.indexOf("?") === -1 ? "?" : "&";
```

Comment:
Also just realized we could use `includes` instead of `indexOf` here and below. I think we used `indexOf` from before includes was as supported

Reply:
Makes sense to me